core/zms/src/main/rdl/Policy.rdli

```diff
@@ -218,4 +219,91 @@ resource AssertionCondition DELETE "/domain/{domainName}/policy/{policyName}/ass
         ResourceError CONFLICT;
         ResourceError TOO_MANY_REQUESTS;
     }
+}
+
+//List policy versions.
+resource PolicyList GET "/domain/{domainName}/policy/{policyName}/version?limit={limit}&skip={skip}" (name=getPolicyVersionList) {
```

Comment:
Got it. Thought that because 3 is a configurable value we might want to keep this open for future updates but if we never planning on having more than a few versions I'll remove it.

Reply:
Fixed